README.md

```diff
@@ -4,13 +4,15 @@ A tool to search for [specification][] violations.  Requires Python 3.
 
 Usage:
 
-    $ mkvirtualenv -p python3 conform
-    $ workon conform
+    $ virtualenv -p python3 conform
```

Comment:
I'm a bit confused with this. `virtualenv` used to be a tool you installed via `pip` or similar. Based on what I can read here https://docs.python.org/3.6/library/venv.html there is another thing like `python3 -m venv /path/to/new/virtual/environment`.